imports/pages/give/review/__tests__/index.js

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable */
+import { Component } from "react";
+import { shallow } from "enzyme";
+import { shallowToJson } from "enzyme-to-json";
+import { reset, startBuffering } from "aphrodite/lib/inject";
+
+import { Template } from "../";
+
+export const sampleData = {
+  "data": {
+    "payment": {
+      "icon": "Visa",
+      "last4": "1112",
```

Comment:
@delianides in master, support for this is removed (just the last4) so your snapshots may need adjusting 👍